docs/articles.bib

```diff
@@ -1,15 +1,16 @@
 #File for all article, arxiv and other web references
+# To add a key, Lastname_year_abbreviatedtitle/firstletteroftitle
```

Comment:
> To add a key, Lastname_year_abbreviatedtitle/firstletteroftitle

@vprusso Would this work for the sake of consistency?

Reply:
We could align with how [Google Scholar does bibtex keys](https://scholar.googleusercontent.com/scholar.bib?q=info:wK_Z0wl5wBoJ:scholar.google.com/&output=citation&scisdr=ClE5Oj9PEMCi-fd4zM0:AFWwaeYAAAAAZWF-1M36GYDDR2lzgkE_gMNj-jo&scisig=AFWwaeYAAAAAZWF-1Ft1DjFeYXIc9LVPOBdjjyU&scisf=4&ct=citation&cd=-1&hl=en), which is mostly what you have there. That is `{lastname}{year}{abbreviatedtitle}` Any opposition to that?

Reply:
I think I am going to create an issue for using Google scholar style for bibtex keys. I'll get to making these changes then.